### DISCUSSION TITLE
Only last output is shown

### DISCUSSION THREAD

**Discussion by bada478** [microsoft/vscode-jupyter#16191] [Questions and Answers]
Hi all!
I was hoping to show different outputs from a python code block; however, it only showed the last output. If it is an expected/designed behavior, is there a way to show all the outputs? 

![image](https://github.com/user-attachments/assets/abc4ddb6-37f5-4a19-8627-4048e49a9363)

**Comment by DonJayamanne** [comment]
To see different outputs for each statement you need to place each statements/code block that produces an output  in a separate cell
